src/main/markdown/doc/latest/tutorial/i18n.md

```diff
@@ -364,13 +366,13 @@ You tell the GWT compiler that StockWatcher now supports the German (de) locale
     *  The English language version loads by default.
 3.  Load the German version.
     *  To the end of the URL, append `&locale=de`
-    *  `http://localhost:8888/StockWatcher.html?gwt.codesvr=127.0.0.1:9997&locale=de
+    *  http://localhost:8888/StockWatcher.html?gwt.codesvr=127.0.0.1:9997&locale=de
     *  All the constants should display in German.
 4.  Enter a stock code with an invalid character.
     *  The error message should display in German.
-    *  Notice, too, that the date and currency formats are localized.
-    *
-        ![screenshot: StockWatcher German UI](images/i18nGerman.png)
+    *  Notice, too, that the date and currency formats are localized.  
```

Comment:
Yes, this is for wrapping the image. Take a look at http://www.gwtproject.org/doc/latest/tutorial/i18n.html .